bottlecap/src/metrics/aggregator.rs

```diff
@@ -43,11 +44,13 @@ struct DistributionMetric {
 }
 
 #[derive(Debug, Clone)]
+#[repr(transparent)]
```

Comment:
What does this do?

Reply:
It hints the rust compiler to compile the struct using a lightweight representation that have just one primitive field.
In this way, the compiled version will be guaranteed to be the same as only that one field, and all the struct methods will be guaranteed to be 0 cost abstraction